.travis.yml

```diff
@@ -1,11 +1,17 @@
 language: elixir
 elixir:
-  - 1.2.3
+  - 1.1.1
+  - 1.2.6
+  - 1.3.2
 otp_release:
-  - 18.2
+  - 18.3
+  - 19.0
 sudo: false
 env:
   - MIX_ENV=test STRIPE_SECRET_KEY=non_empty_secret_key_string
 script: mix coveralls.travis
 after_script:
   - mix inch.report
+exclude:
+  - otp_release: 19.0
+    elixir: 1.1.1
```

Comment:
@JoshSmith Travis builds a matrix with values from `otp_releases` as rows and values from `elixir` as columns. Then it runs the test for each "cell", e.g. a pair of elixir/otp_release.

Using `exclude` we can "cross out" cells for which we don't want Travis to run.